vendor.conf

```diff
@@ -8,7 +8,7 @@ github.com/boltdb/bolt master
 github.com/buger/goterm 2f8dfbc7dbbff5dd1d391ed91482c24df243b2d3
 github.com/containerd/cgroups 77e628511d924b13a77cebdc73b757a47f6d751b
 github.com/containerd/continuity master
-github.com/containernetworking/cni v0.6.0
+github.com/containernetworking/cni ffc1e7b446d795f885311edf393c6209c32855c6
```

Comment:
Can we get a tag here, instead of a straight commit? Is there a tag we can use?

Reply:
no

Reply:
ok fixed